.github/renovate.json

```diff
@@ -207,6 +207,16 @@
       ],
       "depNameTemplate": "mcr.microsoft.com/playwright",
       "datasourceTemplate": "docker"
+    },
+    {
+      "customType": "regex",
+      "fileMatch": [
+        "optimize/docker/test/docker-labels.golden.json"
+      ],
+      "matchStrings": [
+        "(?<depName>docker\\.io\\/library\\/alpine):(?<currentValue>\\d+\\.\\d+\\.\\d+)"
+      ],
+      "datasourceTemplate": "docker"
```

Comment:
nice that you added this 👍 

Just wondering, will this file continue to exist with the single app changes?